x-pack/plugins/canvas/public/components/hooks/workpad/use_incoming_embeddable.ts

```diff
@@ -8,10 +8,11 @@
 import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { fromExpression } from '@kbn/interpreter';
+import { ErrorStrings } from '../../../../i18n';
 import { CANVAS_APP } from '../../../../common/lib';
 import { decode, encode } from '../../../../common/lib/embeddable_dataurl';
 import { CanvasElement, CanvasPage } from '../../../../types';
-import { useEmbeddablesService, useLabsService } from '../../../services';
+import { useEmbeddablesService, useLabsService, useNotifyService } from '../../../services';
 // @ts-expect-error unconverted file
 import { addElement } from '../../../state/actions/elements';
```

Comment:
You can import `fetchAllRenderables` from here.

```suggestion
import { addElement, fetchAllRenderables } from '../../../state/actions/elements';
```